### DISCUSSION TITLE
HanumanInstitute.MvvmDialogs Now Released!

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#8107] [General]
One aspect that is a hell in Avalonia or whenever using MVVM is managing dialogs. I've expanded upon [FantasticFiasco/mvvm-dialogs](https://github.com/FantasticFiasco/mvvm-dialogs) to provide a cross-platform solution and just released the first official version!

[HanumanInstitute.MvvmDialogs for Avalonia](https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs)

I'm quite proud of how this library came out.

Happy coding!! :smiley:

**Comment by timunie** [comment]
Can we add this to Awsome Avalonia please? 🙏

**Reply by mysteryx93** [reply]
Sure!
